code/modules/shuttle/cas_plane.dm

```diff
@@ -42,6 +45,13 @@
 	if(occupant)
 		to_chat(occupant, span_notice("CAS laser detected. Target: [AREACOORD_NO_Z(incoming_laser)]"))
 
+/obj/structure/caspart/caschair/proc/cas_usable(datum/source)
+	SIGNAL_HANDLER
+	cas_usable++
```

Comment:
? Just make it true.
Also are you sure all these signals are needed? The one labeled crash for example, is that not actually related to the crash gamemode?

Reply:
Made it true.
Also, you'd be right on the crash signal. Every other signal is necessary, however.